src/h_vialib/client.py

```diff
@@ -20,19 +28,9 @@ def __init__(self, url, content_type=None):
         self.url = url
 
         if content_type is None and self._GOOGLE_DRIVE_REGEX.match(url):
-            content_type = "pdf"
-
-        self._content_type = content_type
+            content_type = ContentType.PDF
 
-    @property
-    def is_html(self):
```

Comment:
Just compare with `content_type`. We avoid having a new "is_video" etc for each new content type.

Some clients might use that and need updating. We'll see that on the dependabot PRs.

Reply:
Yep, makes sense as a more scalable interface :+1: